### ISSUE TITLE
Budgie-panel block windows from going to second monitor

### ISSUE THREAD

**Issue by TeenCorn** [solus-project/budgie-desktop#1991]
## Description

When the panel is set in the same direction as the monitor, when trying to move a window towards that monitor, the panel will block it. What ends up happening is this weird state where the window is 75% on the second monitor but 25% is still on the main one.

For example:
![block-issue1](https://user-images.githubusercontent.com/19555605/84729295-36cafc00-af48-11ea-8755-d9a9b5d56261.png)

This happens in all directions.

Also, when trying to use the **Move window one monitor to the \<direction\>** shortcut, the window also gets blocked but this time a portion of the window does not move on to the second monitor. It just slams against the edge of the panel.

## Budgie version

budgie-desktop 10.5.1 (git-bc1f87765258a2b716a2627c613552497f5240f1)

## Operating system

Solus

## Steps to reproduce the issue

Set up the panel in any orientation and setup the second display to be in that direction. Then, grab a window and try to move that window past the panel toward your second display. Do the same with the shortcut.

**Comment by SirRujak** [comment]
Just to add to this a bit, it looks like this only happens when automatically hide is set to never. Both automatic and intelligent mode allow windows to properly cross over the panel to adjacent screens.
Here is an example of it working on automatic:
![Budgie_Panel_1](https://user-images.githubusercontent.com/5132268/84798864-12f7cc80-afc1-11ea-84b7-e9bf315dd284.png)

**Comment by TeenCorn** [comment]
😯 That's an interesting find. I would've used this in the meantime if I could focus the panel by hovering the mouse over the edge of the screen when there is a maximized window. With **automatic** I have to either use the Raven keybind **ctrl+a** or unmaximize all maximized windows to get the panel.

**Comment by JoshStrobl** [comment]
Basically a duplicate of #474 since it is being blocked by the panel.
